config/config_test.go

```diff
@@ -169,7 +170,26 @@ func TestParse(t *testing.T) {
 					max_zoom = 20
 					dont_simplify = true
 					dont_clip = true
-					dont_clean = true`,
+					dont_clean = true
+
+					[[maps.params]]
+					name = "param1"
+					token = "!param1!"
+					type = "string"
+					
+					[[maps.params]]
+					name = "param2"
+					token = "!PARAM2!"
```

Comment:
We could do that, however, the user will need to be aware of the transformation `x` → `!X!`, so they'll have to reference the documentation. Figuring out where a param is coming from also becomes more difficult, since you can't simply search for `!YOUR_PARAM!` anymore.

> Here goes my argument about `reading > writing` again.

So for now I'll leave is as is.